index.html

```diff
@@ -25,10 +25,19 @@
 
             <div class="row right" class="right hide-on-med-and-down">
                 <a href="./SigninAuth.html" class="btn-large waves-effect waves-light deep-orange" id="login-btn">User
-
                     Login</a>
             </div>
+
+            <ul id="dropdown1" class="dropdown-content">
+                <li id="one"><a href="#!">one</a></li>
+                <li id="two"><a href="#!">two</a></li>
+                <li id="three"><a href="#!">three</a></li>
+                <li id="four"><a href="#!">four</a></li>
+                <li id="five"><a href="#!">five</a></li>
+            </ul>
```

Comment:
I'm not quite sure I understand the intention of the one two three etc... This should determine the number of tokens in total?
can't we default it to a static number say 5?
If not, can we try to generate them dynamically in the future, by typing in a number?
This comment is not a merge blocker, more of a feature request.

Reply:
realistically the amount of exercises a child can perform successfully before a break (or a tantrum) is going to vary drastically depending on the kid and the day so I would rather allow the therapist to choose. I can make it an input instead I just didnt want to change the look of our beautiful UI too much